pom.xml

```diff
@@ -48,6 +48,8 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+         <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
The indentation looks wrong here.

Reply:
addressed in 1fe5940d667da0881ad8595be524a396dc69a0e0